File: Models/PickupPoint.cs
namespace Transfers {

    public class PickupPoint {

        public int Id { get; set; }
        public string PickupPointDescription { get; set; }
        public string ExactPoint { get; set; }
        public string Time { get; set; }
        public string UserName { get; set; }

        public int RouteId { get; set; }

        public Route Route { get; set; }
    }

}

File: ClientApp/src/app/pickupPoints/classes/pickupPoint.ts
import { KeyValuePair } from 'src/app/shared/classes/model-keyValuePair';

export class PickupPoint {
    id: number
    description: string
    route: {
        id: number
        abbreviation: string
    }
    exactPoint: string
    time: string
    userName: string
}
